frontend/packages/console-app/src/components/nodes/NodeDetailsConditions.tsx

```diff
@@ -28,20 +28,20 @@ const NodeDetailsConditions: React.FC<NodeDetailsConditionsProps> = ({ node }) =
             </tr>
           </thead>
           <tbody className="pf-v6-c-table__tbody">
-            {_.map(node.status.conditions, (c, i) => (
+            {_.map(node.status?.conditions, (c, i) => (
               <tr className="pf-v6-c-table__tr" key={i}>
                 <td className="pf-v6-c-table__td">
                   <CamelCaseWrap value={c.type} />
                 </td>
                 <td className="pf-v6-c-table__td">{c.status || '-'}</td>
                 <td className="pf-v6-c-table__td">
-                  <CamelCaseWrap value={c.reason} />
+                  <CamelCaseWrap value={c.reason || ''} />
                 </td>
                 <td className="pf-v6-c-table__td">
-                  <Timestamp timestamp={c.lastHeartbeatTime} />
+                  <Timestamp timestamp={c.lastHeartbeatTime || ''} />
                 </td>
                 <td className="pf-v6-c-table__td">
-                  <Timestamp timestamp={c.lastTransitionTime} />
+                  <Timestamp timestamp={c.lastTransitionTime || ''} />
```

Comment:
should we instead allow `null` in the timestamp component? since it already has code to check for nullish values:

```js
  // Check for null. If props.timestamp is null, it returns incorrect date and time of Wed Dec 31 1969 19:00:00 GMT-0500 (Eastern Standard Time)
  if (!props.timestamp) {
    return <div className="co-timestamp">-</div>;
  }
```